src/main/webapp/app/exercises/text/manage/text-exercise/text-exercise-update.component.html

```diff
@@ -138,17 +138,18 @@ <h4 *ngIf="isImport" id="jhi-text-exercise-heading-import" jhiTranslate="artemis
                 </div>
             </div>
         </div>
-        <div class="form-group" *ngIf="!isExamMode">
+        <div class="form-group" *ngIf="(isAthenaEnabled$ | async) && !isExamMode">
```

Comment:
Note for reviewers: This is the feedback suggestions enabled checkbox for text exercises.
![screenshot-2023-09-04_001887](https://github.com/ls1intum/Artemis/assets/9006596/ec4aa118-c19d-4bd3-99e5-6729f16ae4a9)